app/models/concerns/smooch_newsletter.rb

```diff
@@ -5,33 +5,19 @@ module SmoochNewsletter
 
   module ClassMethods
     TeamBotInstallation.class_eval do
-      # Re-create the Sidekiq job
```

Comment:
just making sure I understand: this bit was just scheduling the next job and has now been moved to TiplineNewsletter, right?

Reply:
Correct!